German Wiki Creation (#244)
gridcoin-community/Gridcoin-Site: Wiki/de/index.md

---
title: Gridcoin Wiki
layout: wiki-de
---

## Gridcoin Einführung

> Gridcoin (Ticker: GRC) ist eine dezentrale, mathematische,

Reviewer: I would omit the term "mathematische" here.

Reviewer 2: I know my text sounds a bit strange in some sentences, I tried to translate it to the letter, and not to add or leave anything. But I see, it would be better to brush it up a bit. Feel free to make some changes. :)

open source Kryptowährung (digitaler Vermögenswert). 

> Gridcoin ist, neben seiner Energieeffizienz, auch die
> erste und derzeit einzige Kryptowährung, die Einzelpersonen
> für wissenschafftliche Beiträge belohnt und peer-to-peer 
> Transaktionen kryptographisch vollzieht, ohne dabei auf 
> eine zentrale Instanz zur Verteilung der Belohnungen angewiesen 
> zu sein. Gridcoin ist das erste Blockchain Protokoll welches 
> einen funktionierenden Algorithmus zum Nachweis und Belohnung von 
> geleisteter [BOINC](https://en.wikipedia.org/wiki/Berkeley_Open_Infrastructure_for_Network_Computing)
> (Berkeley Open Infrastructure for Network Computing) Rechenarbeit,
> wobei es sich um quasi nahezu jeglichen Computerprozess handeln 
> kann, liefert.

Reviewer: I would rephrase the sentence. 
```suggestion
Gridcoin ist dank der Verwendung des Proof of Stake Konsensus-Mechanismus sehr energieeffizient. Gridcoin ist die
erste und derzeit einzige Kryptowährung, die Einzelpersonen
dafür Belohnt, dass sie ihre Rechenleistung für wissenschafftliche Forschung zur Verfügung stellen. Dabei werden alle Transaktionen kryptographisch sicher und ohne eine zentrale Instanz abgewickelt.

Nutzer, die ihre Rechenleistung [BOINC Projekten](https://en.wikipedia.org/wiki/Berkeley_Open_Infrastructure_for_Network_Computing)
(Berkeley Open Infrastructure for Network Computing) zur Verfügung stellen, können damit Gridcoins verdienen.
```

Reviewer: Yes, PoS is more energy efficient, because a new hash can only be generated every 16 seconds (or 8?).

Reviewer 3: I think we talk about different things. 
In my Opinion, an energy efficient PC is shut down when not used. But, when this PC is crunching during idle times is consumes much more energy than in idle mode!! 
But the BIG difference is, when a PC is dedicated to mining a Coin, he will consume much more energy, just that is his only task. 
So at least, it isn't "more" energy efficient (the german meaning). It is only.... you do something good when your PC is on power and YOU do nothing with it. 

for explanation: The german energy efficient (energieffiezient) meaning is a very narrow thing, like LED is much more energy efficient than a light bulb. 
But in this case, your PC is running at 100% while you do nothing but you can earn a little bit money and help some science guys. (GRC) vs. your PC is running at 100% just for mining money (ex. BTC). 

Sorry for my long explanation. The **text isn't wrong** and in the last 20 years, I recommended BOINC everybody who run their computers 24/7. So I am on your side, I just don't like the word energy efficient in this meaning.

Author: The idea (at least from what its trying to convey from the English version) is that proof of stake itself isn't what's using up energy. The system of proof of stake doesn't use up much energy and so most of the computations involved in Gridcoin are on BOINC (not a part of proof of stake) meaning that they're useful. This means that overall there's less wasted computation in Gridcoin. 

Maybe this could be phrased differently in German to clarify?

Reviewer: @lucky2d thanks for the clarification. Now I see what you mean.
I will change the text a bit.

Author: @lucky2d can you check over the edited suggestion?

Here is their edited suggestion for reference
```suggestion
Das Erstellen von neuen Blöcken auf der Gridcoin Blockchain ist dank der Verwendung des Proof of Stake Konsensus-Mechanismus sehr energieeffizient. Gridcoin ist die
erste und derzeit einzige Kryptowährung, die Einzelpersonen
dafür Belohnt, dass sie ihre Rechenleistung für wissenschafftliche Forschung zur Verfügung stellen. Dabei werden alle Transaktionen kryptographisch sicher und ohne eine zentrale Instanz abgewickelt.
Nutzer, die ihre Rechenleistung [BOINC Projekten](https://en.wikipedia.org/wiki/Berkeley_Open_Infrastructure_for_Network_Computing)
(Berkeley Open Infrastructure for Network Computing) zur Verfügung stellen, können damit Gridcoins verdienen
```

Reviewer 3: Thank you @RoboticMind. Now, it sounds very good. Just only one typo in line 12. The word "Belohnt" is a verb and therefore it is writte in small letters "belohnt"

Durch die zur Verfügung gestellte Computerkapazitäten und den damit 

> ausgeführte Rechenprozessen bietet Gridcoin der Menschheit einen Nutzen. 

Reviewer: Approved.

> Es gibt keine Einschränkungen für ein einzelnes BOINC Projekt, was zur
> Folge hat, dass so gut wie jeder Computervorgang belohnt werden kann.

Reviewer: ```suggestion
Prinzipiell gibt es keine Einschränkungen, für welche BOINC Projekte Nutzer mit Gridcoins belohnt werden können. Gridcoin nutzt jedoch eine Whitelist und es lassen sich nur mit Projekten auf der Whitelist Gridcoins verdienen. Um ein Projekt auf die Whitelist zu bringen, kann eine dezentrale Abstimmung auf der Blockchain gestartet werden. Ist die Mehrheit der Nutzer dafür, so wird das Projekt in die Whitelist aufgenommen.
```

> Einer Nachricht auf [CryptocurrencyTalk](https://cryptocurrencytalk.com/topic/1416-gridcoin-grc-information-classic-in-retirement/) zufolge, wurde Gridcoin am 16. Oktober 2013 von 
> Rob Halförd gegründet.

Reviewer: ```suggestion
Gridcoin wurde am 16. Oktober 2013 von einem anonymen Programmierer mit dem Pseudonym Rob Halförd gegründet, wie man einer Nachricht auf [CryptocurrencyTalk](https://cryptocurrencytalk.com/topic/1416-gridcoin-grc-information-classic-in-retirement/) entnehmen kann.
```

> Die Coin entschädigt die "Miner" für ihren Beitrag in [BOINC](boinc "wikilink")

Reviewer: approved.

Projekten, welche Fortschritte in Bereichen wie Medizin, Biologie, 
Mathematik, Klimaforschung, Partikel- oder Astrophysik erzielen können
und somit erfolgreiche wissenschaftliche Arbeit für die Menschheit zur Folge 

> haben können.

Reviewer: I'd write "Projekten für Medizin, Biologie, Mathematik, Klimaforschung, Partikel- oder Astrophysik."

> Projekten, welche Fortschritte in Bereichen wie Medizin, Biologie, 
> Mathematik, Klimaforschung, Partikel- oder Astrophysik erzielen können
> und somit erfolgreiche wissenschaftliche Arbeit für die Menschheit zur Folge 
> haben können.

Reviewer 3: ```suggestion
Projekten wie Medizin, Biologie, 
Mathematik, Klimaforschung, Partikel- oder Astrophysik.
Die "Miner" unterstützen damit erfolgreich wissenschaftliche Forschung.
```

> haben können.

Author: Should I implement this suggestion with `Projekten für` or `Projekten wie`?

> haben können.

Reviewer: I'm for "Projekten für".

> haben können.

Reviewer 3: "Projekten für" sound better

> Gridcoin entlohnt die Nutzer für die Bereitstellung ihrer Computer, 
> indem es die Proof-of-Stake Methode anwendet. Diese ist im Gegensatz 

Reviewer: ```suggestion
Neue Blöcke werden in Gridcoin durch den Proof of Stake Mechanismus generiert.
Diese ist im Gegensatz 
```

> zum rechenintensiven und somit hitzeerzeugenden Proof-of Work Algorithmus

Reviewer: I'd write Proof of Work instead of Proof-of Work.
It's written like that on the german wikipedia page (https://de.wikipedia.org/wiki/Proof_of_Work)

> zum rechenintensiven und somit hitzeerzeugenden Proof-of Work Algorithmus

Reviewer 3: ```suggestion
zum rechenintensiven und somit hitzeerzeugenden Proof of Work Algorithmus
```

> deutlich ressourcenschonender und energieeffizient. Gridcoin unterscheidet 

Reviewer: approved

> deutlich ressourcenschonender und energieeffizient. Gridcoin unterscheidet 
> sich von anderen wissenschaftlichen Kryptowährungen durch seine Fähigkeit, 
> viele verschiedene Arten von Projekten unterstützen zu können. Die Zahl derer 

Reviewer: ```suggestion
deutlich ressourcenschonender und energieeffizient. Gridcoin unterscheidet 
sich von anderen Kryptowährungen mit dem Ziel das Mining für sinnvolle Berechnungen zu nutzen, durch seine Fähigkeit, 
viele verschiedene Arten von Projekten unterstützen zu können. Die Zahl derer 
```

> viele verschiedene Arten von Projekten unterstützen zu können. Die Zahl derer 

Author: @Pythonix I think you missed this suggestion in your review because it's marked as outdated because the number of lines changed somewhere else, but the text here didn't change. Is this good to use?

wird im Grunde nur durch die momentan aktiven BOINC Projekte und durch die 
"project whitelist", über welche die Community abstimmt, begrenzt.

> Es gab in der Vergangenheit einen Wechsel vom originalen [Gridcoin-Classic](gridcoin-classic "wikilink")

Reviewer: approved

> zu "Gridcoin-Research", der eigentlichen Gridcoin. Gridcoin-Classic nutzte einen 

Reviewer: approved

> Proof-of-Work Hybriden, wodurch die Computer die halbe Rechenzeit mit "Hashing" beschäftigte,
> während sie die andere Hälfte ihren Beitrag zur Wissenschaft leisteten; Zwar eine
> Verbesserung zu Bitcoin, es wurden aber immer noch Computerressourcen vergeudet.

Reviewer 3: ```suggestion
Proof-of-Work Hybriden, wodurch die Computer die Hälfte ihrer Rechenzeit ihren Beitrag zur Wissenschaft leisteten 
 und die andere Hälfte mit "Hashing" verbrachten. Zwar eine Verbesserung zu Bitcoin, aber es wurden immer noch Computerressourcen vergeudet.
```

Reviewer: Here as well. I'd write Proof of Work instead of Proof-of-Work.

Reviewer 3: ```suggestion
Proof of Work Hybriden, wodurch die Computer die halbe Rechenzeit mit "Hashing" beschäftigte,
während sie die andere Hälfte ihren Beitrag zur Wissenschaft leisteten; Zwar eine
Verbesserung zu Bitcoin, es wurden aber immer noch Computerressourcen vergeudet.
```

Die BOINC Arbeit wurde nur dazu genutzt, um die Beihilfe zu messen, welche mit dem

> Block geschürft wurde. 

Reviewer: I don't understand this part. Can probably be deleted.

> Die BOINC Arbeit wurde nur dazu genutzt, um die Beihilfe zu messen, welche mit dem
> Block geschürft wurde. 

Reviewer 3: ```suggestion
Die BOINC Arbeit wurde nur dazu genutzt, um den Anteil zu berechnen, welcher mit dem
Block geschürft wurde. 
```

Reviewer: I still don't understand this part.

Reviewer 3: Im Prinzip geht es in diesem Abschnitt darum zu erklären dass das crunchen von Boinc Projekten einen Anteil daran hat wieviel man als Gegenleistung in GRC erhält. 
But at least I will follow Pythonix, i didn't make a real sense in german, so delete it.

> Gridcoin-Research macht verschlüsseltes Mining und Proof-of-Work hashing obsolet.

Reviewer: ```suggestion
Gridcoin-Research macht Proof of Work überflüssig.
```

Mit dem Wechsel zu Gridcoin-Research und damit zum Proof-of-Stake Algorithmus, wird

> nun nahezu die ganze Energie der wissenschaft übergeben. Die Rechenarbeit die jetzt

Reviewer: ```suggestion
nun nahezu die ganze Energie für wissenschaftliche Berechnungen verwendet. Die Rechenarbeit, die jetzt
```

> nun nahezu die ganze Energie der wissenschaft übergeben. Die Rechenarbeit die jetzt

Reviewer 3: ```suggestion
nun nahezu die vollständige Rechenleistung der Wissenschaft zur Verfügung gestellt. Der Anteil der Berechnung welcher ```

Reviewer: approved

zur Auffrechterhaltung der Blockchain notwendig ist, ist minimal.

## Metrics

**[Gridcoin-Classic](gridcoin-classic "wikilink") - Retired**

  - Start: 16. Oktober 2013
  - Ende: 20. April 2015
  - Von: Rob Halförd
  - Algorithmus: Scrypt Proof-of-Work
  - Angebot: 40 Million Approx.
  - Block Zeit: 2.5 Minutes
  - Retarget: Every 30 Minutes
  - Belohnung: variabel
  - Vorgefertigt: 0%

**Gridcoin-Research Proof-of-Stake**

  - Start: 11. Oktober 2014
  - Von: Rob Halförd
  - Algorithmus: Proof of Stake. Belohnung basierend auf 10 GRC + die geleistete
    Forschungsarbeit auf BOINC
  - Angebot: 10 GRC je neuem Block (~9600 GRC pro Tag) und ~28000 neue GRC für
    Forschungsbelohnungen
  - Block Zeit: 90 Seconds
  - Vorgefertigt: 0% - Die anfängliche Verteilung basierte auf Proof-of-Burn von
    Gridcoin Classic - Research Foundation Block = 340,569,880

## P2P

Gridcoin ermöglicht peer to peer (P2P) Transaktionen weltweit und nahezu in
Echtzeit, wofür nur sehr geringe Gebühren fällig werden.

> Der Code ist open source und erlaubt es jedem am Erfolg und in der 
> Entwicklung teilzuhaben. Open Source ist in der Regel sicherer, da

Reviewer: ```suggestion
Der Code ist open source und erlaubt es jedem, an der Entwicklung mitzuarbeiten. Open Source ist in der Regel sicherer, da
```

> jeder den Code einsehen kann und ihn im Bedarf verbessern kann.

Reviewer 3: ```suggestion
jeder den Code einsehen kann und ihn bei Bedarf verbessern kann.
```

Reviewer: approved

> Der Erfolg von BOINC beruht hauptsächlich auf seinem Credit System, welches
> einen Wettbewerb zwischen den Nutzern ermöglicht. Gridcoin erweitert dies,
> indem es einen austauschbaren Maßstab hinzufügt. Teilnehmer können nun
> ihren eigenen Beitrag einsehen und haben mit der Blockchain ein Dokument für
> den wissenschaftlichen Fortschritt.

Reviewer: ```suggestion
BOINC nutzt ein Credit System, welches Nutzer dazu anspornt, Rechenleistung zu spenden. Die Credits erlauben es dann, sich mit anderen Nutzern zu vergleichen. Gridcoin fügt dem ganzen nun noch einen finanziellen Anreiz hinzu. Je mehr Rechenleistung jemand spendet, desto mehr Gridcoins verdient er damit.
```

## Vergleich zu Bitcoin

Bitcoin verbraucht geschätzt 59.31 TWh, laut
[Digiconomist](https://digiconomist.net/bitcoin-energy-consumption).
Diese Zahl wird zwar viel diskutiert, sie zeigt aber sicherlich, dass

> eine Menge Energie von Bitcoin's SHA-256 Algorithmus verbraucht wird.
> Diese Energie könnte für weitaus sinnvollere Kalkulationen verwendet werden.

Reviewer: ```suggestion
eine Menge Energie von Bitcoins SHA-256 Algorithmus verbraucht wird.
Diese Energie könnte für weitaus sinnvollere Berechnungen verwendet werden.
```

Reviewer 3: I would suggest to use "Rechenleistung" instead of "Energie"

Reviewer: Changed =)

> Gridcoin basiert auf Dem Protokoll von Bitcoin, sodass es effektiv mit
> normaler "Verbraucher-Hardware" geschürft werden kann.

Reviewer: ```suggestion
Gridcoin basiert auf dem Protokoll von Bitcoin und kann mit
normaler "Verbraucher-Hardware" geschürft werden.
```

Es ist somit unwahrscheinlich, dass diese Hardware künftig durch spezialisierte
ASIC Mining Chips ersetzt wird.

> Gridcoin Research basiert auf dem Proof-of-Stake Algorithmus, der deutlich
> energieeffizienter ist. Energieeffizienz wurde bereits bei der Veröffentlichung
> von Peercoin, der ersten Proof-of-Stake Coin, verkündet. Prof-of-Stake erlaubt es
> den Schürfern ihre Computerressourcen ganz der Wissenschaft zu widmen, anstatt
> sie zur Unterhaltung der Blockchain zu verschwenden.

Reviewer: ```suggestion
Gridcoin Research basiert auf dem Proof of Stake Algorithmus, der deutlich
energieeffizienter ist als der Proof of Work Algorithmus. Nutzer können zusätzlich Gridcoins verdienen, wenn sie mit ihrer Rechenleistung BOINC Projekte unterstützen.
```

<!-- end list -->

  - Quellcode:
      - [Gridcoin-Research](https://github.com/gridcoin-community/Gridcoin-Research)
      - [Gridcoin-Classic (**retired**)](https://github.com/gridcoin/Gridcoin-master)

<!-- end list -->

  - Weitere Quellen:
      - [Gridcoin's GitHub community organization](https://github.com/gridcoin-community)
          - [Website Development](https://github.com/gridcoin-community/Gridcoin-Site)
          - [Gridcoin Tasks](https://github.com/gridcoin-community/Gridcoin-Tasks)
          - [Marketing images](https://github.com/gridcoin-community/Gridcoin-Marketing)
          - [Fan Site (Gridcoin Science)](https://gridcoin.science)

<!-- end list -->

  - Downloads:
      - [Github](https://github.com/gridcoin-community/Gridcoin-Research/releases)
      - [Launchpad](https://launchpad.net/gridcoin)
      - [AUR Qt](https://aur.archlinux.org/packages/gridcoinresearch-qt)
      - [AUR daemon](https://aur.archlinux.org/packages/gridcoinresearchd/)

<!-- end list -->

  - Handelsplätze:
      - [Txbit](https://txbit.io/Trade/GRC/BTC)
      - [SouthXchange](https://www.southxchange.com/Market/Book/GRC/BTC)

<!-- end list -->

>   - Weitere Trading Möglichkeiten:

Reviewer 3: ```suggestion
  - Weitere Möglichkeiten zum Tauschen von GRC:
```

Reviewer: I'd write "Handelnd" instead of tauschen.

"Weitere Möglichkeiten zum Handeln von GRC:"

Reviewer 3: ```suggestion
  - Weitere Möglichkeiten zum Handeln von GRC:
```

      - [Cryptocurrencytalk macOS](https://cryptocurrencytalk.com/topic/13139-os-x-builds-feedback-bug-reporting/)
      - [Bitcointalk](https://bitcointalk.org/index.php?topic=324118.0)
      - [Steemit](https://steemit.com/created/gridcoin)
      - [Steemit Community](https://steemit.com/created/hive-161364)
      - [Reddit](https://reddit.com/r/gridcoin)
      - [Reddit (ru)](https://reddit.com/r/russiangridcoin)
      - [Gridcoin Telegram](https://t.me/gridcoin)
      - [Discord](https://discord.me/page/gridcoin)
      - [Slack](https://join.slack.com/t/teamgridcoin/shared_invite/enQtMjk2NTI4MzAwMzg0LTUzMmY0YjdiNzYxYzQ0MDc3MGE1NjQ3Nzg2NWMzZTUzMjAwZjdhMWI1YWUzMDE4YzQyZjVjMjMzOTc1M2RmMmM/)
      - IRC: [#gridcoin](https://kiwiirc.com/client/irc.freenode.net:6697/#gridcoin) and [#gridcoin-help](https://kiwiirc.com/client/irc.freenode.net:6697/#gridcoin-help) on freenode.

<!-- end list -->

  - Soziale Medien:
      - [Twitter](https://twitter.com/GridcoinNetwork)
      - [YouTube](https://www.youtube.com/c/GridcoinNetwork)
      - [Tumblr](https://gridcoin.tumblr.com/)
      - [Facebook](https://facebook.com/gridcoins)

>   - Charts:

Reviewer 3: ```suggestion
  - Übersichten, Statistiken und Marktkapitalisierung:
```

Reviewer: Approved

## Gridcoin Einführung

> Gridcoin (Ticker: GRC) ist eine dezentrale,

Reviewer 3: I think this comma can be deleted

Reviewer: approved
